examples/data-objects/monaco/src/view.ts

```diff
@@ -51,25 +49,8 @@ const defaultCompilerOptions = {
     allowNonTsExtensions: true,
 };
 
-/**
- * Component for using the Monaco text editor.
- */
-export class MonacoRunner extends DataObject implements IFluidHTMLView {
+export class MonacoRunnerView implements IFluidHTMLView {
```

Comment:
So this eventually can turn into react component. It seems like there are no other dependences in the way the view is consumed ? Maybe ContainerViewRuntimeFactory expecting IFluidHTMLView ?

Reply:
Yep this can become a React component eventually.  ContainerViewRuntimeFactory just expects something that MountableView can mount, which currently includes either IFluidHTMLView or React components.